tools/ReadConsoleInputStream/ReadConsoleInputStreamDemo.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RootNamespace>Nivot.Terminal</RootNamespace>
```

Comment:
this is technically incorrect \\_\(shrug)\_/

Reply:
The namespace and unsafe block. Got it.